templates/docker-entrypoint.sh

```diff
@@ -150,6 +150,10 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
     echo "\$config['spellcheck_uri'] = '${ROUNDCUBEMAIL_SPELLCHECK_URI}';" >> config/config.docker.inc.php
   fi
 
+  if [[ "$ROUNDCUBEMAIL_PLUGINS" = *enigma* ]]; then
+    echo "$config['enigma_pgp_homedir'] = '/var/roundcube/enigma';" >> config/config.docker.inc.php
```

Comment:
Maybe add a line in the Readme about this directory (section "Persistent data")?

Reply:
Very good point, thank you!